.goreleaser.import.yml

```diff
@@ -10,11 +10,15 @@ builds:
     goarch:
       - amd64
       - arm64
+    env:
+      - CGO_ENABLED=0
+    ldflags:
+      - '-s -w -X main.version={{.Version}} -X main.commit={{.Commit}}'
```

Comment:
What do these do?

Reply:
`CGO_ENABLED=0` disables the Go interface to c libraries

`-s`: Omits the symbol table and debug information.
`-w`: Omits the DWARF symbol table.

the other ones inject the version and commit - i removed as we don't need that here